wagtailnhsukfrontend/templates/wagtailnhsukfrontend/do_list.html

```diff
@@ -1,5 +1,11 @@
 <div class="nhsuk-do-dont-list">
-  <h{{ heading_level }} class="nhsuk-do-dont-list__label">Do</h{{ heading_level }} >
+  <h{{ heading_level }} class="nhsuk-do-dont-list__label">
```

Comment:
You can use the "default" tag here instead https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#default

` {{ label|default:"Do" }}`

Reply:
Nice one, I didn't think of that.